src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx

```diff
@@ -496,13 +496,16 @@ export default class QueryTabComponent extends React.Component<IQueryTabComponen
   };
 
   public onChangeContent(newContent: string): void {
+    // The copilot store's active query takes precedence over the local state,
```

Comment:
I would remove these comments as well.

Reply:
I did think this was a useful one because the fact that the copilot store takes precedence is not clear when reading _this_ method. In fact, it took some debugging to realize that it was this state update ordering that was causing one of the problems I was seeing. So without the comment, someone might reorder the state updates and introduce a difficult-to-detect bug. Happy to remove it though, if folks still think it's unnecessary.